vip-api/src/main/java/fr/insalyon/creatis/vip/api/business/DataApiBusiness.java

```diff
@@ -281,12 +282,13 @@ private void checkDownloadPermission(String path) throws ApiException {
     private void checkPermission(String path, LFCAccessType accessType)
             throws ApiException {
         try {
-            lfcPermissionBusiness.checkPermission(
-                currentUserProvider.get(), path, accessType);
+            if ( ! lfcPermissionBusiness.isLFCPathAllowed(
+                currentUserProvider.get(), path, accessType, true)) {
```

Comment:
Indeed it is for security reasons. It could be useful to have an API use for admin purposes but I would first add a more robust authentication method.